### DISCUSSION TITLE
Getting Started with WiX

### DISCUSSION THREAD

**Discussion by drusellers** [wixtoolset/issues#8634] [Questions]
Hello Team :wave:,

I was reading through this page, https://wixtoolset.org/docs/intro/
and was wondering if there is any other documentation that takes someone from "zero to hero"?

Specifically, I would love to see a small "Quick Start" that works with dotnet core, that uses WiX to package up a simple command line application into an MSI. This would greatly help me on my journey.

It's possible this exists already, and I just need some guidance.

TY

-d

**Comment by robmen** [comment]
[WiX Toolset v4 and v5 Tutorial](https://www.firegiant.com/docs/wix/tutorial/)

**Reply by drusellers** [reply]
Ah, that is using Heatwave and a VS plugin - being more of a CLI kid  - is there anything that is more "bare bones"? If not, totally understand just wanted to ask. 

In that tutorial - is there a completed version in GitHub that I could look at. I would love to see all the files laid out in a repository that I can poke around at.

TY !!!

**Reply by robmen** [reply]
Try the [Deployment Dojo](https://deployment dojo.com/).

**Reply by tjmoore** [reply]
I appreciate the effort that's gone into video tutorials, but that's a serious training investment to sit through 50+ 30min to 1hr episodes!

As for the Firegiant tutorial, it bails out at the very bare minimum and doesn't cover any UI steps. I've dabbled with v3 and could go with what I've done before, but I don't know how much ports across and would rather start clean with v5 and use it properly, not use things I then find out are deprecated.

+1 on complete examples in GitHub

**Comment by rsmart8452** [comment]
n13org has tutorials for basic installers - probably what you're looking for. https://github.com/n13org

Don't be intimidated by Deployment Dojo's volume. I suggest using it as a reference manual. Pull up the list of episodes and check out what you fancy. It's not a soap opera! If you prefer learning by watching, the guys do a phenomenal job.
